### PR TITLE
Standardise the I/O and time BBEs

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$title

### CODE REVIEW

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

File: examples/csv-io/csv_io.bal [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
     io:println(readCsv);
 
-    // Write the given content stream to a CSV file.
+    // Writes the given content stream to a CSV file.
     check io:fileWriteCsvFromStream(csvFilePath2, csvContent.toStream());
     // If the write operation was successful, then, perform a read operation to read the CSV content as a stream.
     stream<string[], io:Error?> csvStream = check
                                         io:fileReadCsvAsStream(csvFilePath2);
-    // Loop through the stream and print the content.
+    // Iterates through the stream and print the content.
```
Comment (praneesha) [inline examples/csv-io/csv_io.bal:22]:
```suggestion
    // Iterates through the stream and prints the content.
```

File: examples/strings-io/strings_io.bal [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
     io:println(readLines);
 
-    // Write the given stream of lines to a file.
+    // Writes the given stream of lines to a file.
     check io:fileWriteLinesFromStream(textFilePath3, lines.toStream());
     // If the write operation was successful, then, perform a read operation to read the lines as a stream.
     stream<string, io:Error?> lineStream = check
                                     io:fileReadLinesAsStream(textFilePath3);
-    // Loop through the stream and print the content.
+    // Iterates through the stream and print the content.
```
Comment (praneesha) [inline examples/strings-io/strings_io.bal:29]:
```suggestion
    // Iterates through the stream and prints the content.
```

File: examples/time/time.bal [hunk 3]
```diff
@@ -2,7 +2,7 @@ import ballerina/io;
 import ballerina/time;
 
 public function main() returns error? {
-    // Get the current instant of the system clock(seconds from the epoch of
+    // Gets the current instant of the system clock(seconds from the epoch of
```
Comment (praneesha) [inline examples/time/time.bal:5]:
```suggestion
    // Gets the current instant of the system clock (seconds from the epoch of
```

File: examples/time/time.bal [hunk 4]
```diff
@@ -2,7 +2,7 @@ import ballerina/io;
...
     // 1970-01-01T00:00:00). The returned `time:Utc` value represents seconds
     // from the epoch with nanoseconds precision.
     // The `time:Utc` is a tuple with [int, decimal]. The first member of the
```
Comment (praneesha) [inline examples/time/time.bal:8]:
```suggestion
    // The `time:Utc` is a tuple with `[int, decimal]`. The first member of the
```

File: examples/time/time.bal [hunk 5]
```diff
@@ -14,7 +14,7 @@ public function main() returns error? {
     io:println(`Current timestamp nanoseconds as a
     fraction: ${currentUtc[1]}s`);
 
-    // Return seconds from some unspecified epoch. The returned `seconds` has
+    // Returns seconds from some unspecified epoch. The returned `seconds` has
```
Comment (praneesha) [inline examples/time/time.bal:17]:
```suggestion
    // Returns seconds from some unspecified epoch. The returned `seconds` have
```

File: examples/time/time.bal [hunk 6]
```diff
@@ -14,7 +14,7 @@ public function main() returns error? {
...
     // the nanoseconds precision which represents using the fractional part.
```
Comment (praneesha) [inline examples/time/time.bal:18]:
```suggestion
    // the nanoseconds precision, which represents using the fractional part.
```

File: examples/time/time.bal [hunk 7]
```diff
@@ -34,12 +34,12 @@ public function main() returns error? {
     time:Seconds utcDiff = time:utcDiffSeconds(currentUtc, utc1);
     io:println(`UTC diff: ${utcDiff}`);
 
-    // Check the given `time:Date` value is valid or not.
+    // Checks the given `time:Date` value is valid or not.
```
Comment (praneesha) [inline examples/time/time.bal:37]:
```suggestion
    // Checks if the given `time:Date` value is valid or not.
```

File: examples/time/time.bal [hunk 8]
```diff
@@ -34,12 +34,12 @@ public function main() returns error? {
...
     time:Date date = {year: 1994, month: 11, day: 7};
     boolean isValidDate = (time:dateValidate(date) is ()) ? true : false;
     io:println(string `Is valid date: ${isValidDate}`);
 
-    // Return the day of week as a number starting from 0 to 6 inclusive.
+    // Returns the day of week as a number starting from 0 to 6 inclusive.
```
Comment (praneesha) [inline examples/time/time.bal:42]:
```suggestion
    // Returns the day of the week as a number starting from 0 to 6 inclusive.
```

File: examples/xml-io/xml_io.bal [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
 public function main() returns @tainted error? {
-    // Initialize the XML file path and content.
+    // Initializes the XML file path and content.
     string xmlFilePath = "./files/xmlFile.xml";
     xml xmlContent = xml `<book>The Lost World</book>`;
 
-    // Write the given XML to a file.
+    // Writes the given XML to a file.
     check io:fileWriteXml(xmlFilePath, xmlContent);
     // If the write operation was successful, then, perform a read operation to read the XML content.
```
Comment (praneesha) [inline examples/xml-io/xml_io.bal:10]:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the XML content.
```

Comment (BuddhiWathsala) [reply examples/xml-io/xml_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 8 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 7 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 6 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 5 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 4 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/time/time.bal (hunk 3 above)
Comment (BuddhiWathsala) [reply examples/time/time.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/strings-io/strings_io.bal (hunk 2 above)
Comment (BuddhiWathsala) [reply examples/strings-io/strings_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/csv-io/csv_io.bal (hunk 1 above)
Comment (BuddhiWathsala) [reply examples/csv-io/csv_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/7474f8ba8d119904de969e213366674d1dec8383

File: examples/csv-io/csv_io.bal [hunk 10]
```diff
@@ -1,25 +1,25 @@
...
     string csvFilePath1 = "./files/csvFile1.csv";
     string csvFilePath2 = "./files/csvFile2.csv";
     string[][] csvContent = [["1", "James", "10000"], ["2", "Nathan", "150000"],
     ["3", "Ronald", "120000"], ["4", "Roy", "6000"],
     ["5", "Oliver", "1100000"]];
 
-    // Write the given content string[][] to a CSV file.
+    // Writes the given content string[][] to a CSV file.
     check io:fileWriteCsv(csvFilePath1, csvContent);
     // If the write operation was successful, then, perform a read operation to read the CSV content as a string array of arrays.
```
Comment (praneesha) [inline examples/csv-io/csv_io.bal:13]:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the CSV content as a string array of arrays.
```

File: examples/json-io/json_io.bal [hunk 11]
```diff
@@ -13,7 +13,7 @@ public function main() returns @tainted error? {
             "codes": ["4", "8"]
         }};
 
-    // Write the given JSON to a file.
+    // Writes the given JSON to a file.
     check io:fileWriteJson(jsonFilePath, jsonContent);
     // If the write operation was successful, then, perform a read operation to read the JSON content.
```
Comment (praneesha) [inline examples/json-io/json_io.bal:18]:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the JSON content.
```

File: examples/strings-io/strings_io.bal [hunk 12]
```diff
@@ -1,32 +1,32 @@
...
     string textFilePath2 = "./files/textfile2.txt";
     string textFilePath3 = "./files/textfile3.txt";
     string textContent = "Ballerina is an open source programming language.";
     string[] lines = ["The Big Bang Theory", "F.R.I.E.N.D.S",
     "Game of Thrones", "LOST"];
 
-    // Write the given string to a file.
+    // Writes the given string to a file.
     check io:fileWriteString(textFilePath1, textContent);
     // If the write operation was successful, then, read the content as a string.
```
Comment (praneesha) [inline examples/strings-io/strings_io.bal:14]:
```suggestion
    // If the write operation was successful, then, reads the content as a string.
```

File: examples/strings-io/strings_io.bal [hunk 13]
```diff
@@ -1,32 +1,32 @@
...
     string readContent = check io:fileReadString(textFilePath1);
     io:println(readContent);
 
-    // Write the given array of lines to a file.
+    // Writes the given array of lines to a file.
     check io:fileWriteLines(textFilePath2, lines);
     // If the write operation was successful, then, perform a read operation to read the lines as an array.
```
Comment (praneesha) [inline examples/strings-io/strings_io.bal:20]:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the lines as an array.
```

File: examples/strings-io/strings_io.bal [hunk 14]
```diff
@@ -1,32 +1,32 @@
...
     string[] readLines = check io:fileReadLines(textFilePath2);
     io:println(readLines);
 
-    // Write the given stream of lines to a file.
+    // Writes the given stream of lines to a file.
     check io:fileWriteLinesFromStream(textFilePath3, lines.toStream());
     // If the write operation was successful, then, perform a read operation to read the lines as a stream.
```
Comment (praneesha) [inline examples/strings-io/strings_io.bal:26]:
```suggestion
    // If the write operation was successful, then, performs a read operation to read the lines as a stream.
```

Comment (BuddhiWathsala) [reply examples/strings-io/strings_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84

File: examples/strings-io/strings_io.bal (hunk 13 above)
Comment (BuddhiWathsala) [reply examples/strings-io/strings_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84

File: examples/strings-io/strings_io.bal (hunk 12 above)
Comment (BuddhiWathsala) [reply examples/strings-io/strings_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84

File: examples/json-io/json_io.bal (hunk 11 above)
Comment (BuddhiWathsala) [reply examples/json-io/json_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84

File: examples/csv-io/csv_io.bal (hunk 10 above)
Comment (BuddhiWathsala) [reply examples/csv-io/csv_io.bal]: Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1674/commits/e7dc5ab61d2e03737d6158d4fd59a02371e55d84

**Review by daneshk — APPROVED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**